### PR TITLE
isbn-verifier: add new exercise

### PR DESCRIPTION
**Pull request by guygastineau**
Thought it would be nice to get another exercise up on the track :smile:

[canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/isbn-verifier/canonical-data.json)

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: exercises/isbn-verifier/isbn_verifier_test.sh [hunk 1]
```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bats
```
Comment (budmc29) [inline exercises/isbn-verifier/isbn_verifier_test.sh]: Please use `bash` instead of `bats`

**Comment by budmc29** [comment]
Should be good to go after you update the `shebang`

**Comment by guygastineau** [comment]
Oh yeah, than you

**Comment by guygastineau** [comment]
Fixed the she-bang.

Since you approved it pending that fix @budmc29 I will go ahead and squash merge this.

Thank you for catching that ;)

**Pull request merged**